specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/elasticsan.json

```diff
@@ -901,6 +901,20 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "x-ms-delete-snapshots",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "description": "Optional, used to delete snapshots under volume."
+          },
+          {
+            "name": "x-ms-force-delete",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "description": "Optional, used to delete volume if active sessions present."
```

Comment:
As the type is string, please add allowed values.

Reply:
Updated to Boolean

Reply:
I looked around more, i don't think Header values can have any other data type then string. Let me know if i am missing anything here.

Reply:
Changed to have true/ false allowed values only.